src/Forms/Main.cs

```diff
@@ -6087,29 +6087,29 @@ private void ReloadFromSourceView()
                 if (!string.IsNullOrWhiteSpace(textBoxSource.Text))
                 {
                     SubtitleFormat format = GetCurrentSubtitleFormat();
-                    var list = textBoxSource.Lines.ToList();
-                    format = new Subtitle().ReloadLoadSubtitle(list, null, format);
+                    var lines = new List<string>(textBoxSource.Lines);
+                    format = new Subtitle().ReloadLoadSubtitle(lines, null, format);

```

Comment:
2n => n

Reply:
Ran some tests (1000 iterations in release mode without VS) and `textBoxSource.Lines.ToList();` and `new List<string>(textBoxSource.Lines)` are *very* close - ToList seems a little faster with large subtitles and new List seems a little faster with small subtitles, so not really a clear winner IMO.